### YamlMime:ManagedReference
items:
- uid: System.Data.Services.Client.DataServiceQueryException
  id: DataServiceQueryException
  children:
  - System.Data.Services.Client.DataServiceQueryException.#ctor
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  - System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  - System.Data.Services.Client.DataServiceQueryException.Response
  langs:
  - csharp
  name: DataServiceQueryException
  nameWithType: DataServiceQueryException
  fullName: System.Data.Services.Client.DataServiceQueryException
  type: Class
  summary: "Exceção que indica um erro ocorreu ao carregar o valor da propriedade do serviço de dados."
  remarks: "O chamador deve inspecionar <xref:System.Data.Services.Client.DataServiceQueryException.Response%2A>para obter os resultados de exceção.</xref:System.Data.Services.Client.DataServiceQueryException.Response%2A>"
  syntax:
    content: >-
      [System.Diagnostics.DebuggerDisplay("{Message}")]

      public sealed class DataServiceQueryException : InvalidOperationException
  inheritance:
  - System.InvalidOperationException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor
  id: '#ctor'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException()
  nameWithType: DataServiceQueryException.DataServiceQueryException()
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException()
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;> </xref> classe com uma mensagem fornecida pelo sistema que descreve o erro."
  remarks: "Este construtor inicializa o <xref:System.Exception.Message%2A>propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro.</xref:System.Exception.Message%2A> Esta mensagem leva em conta a cultura atual do sistema.       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro localizada. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException ();
    parameters: []
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;> </xref> classe com uma mensagem especificada que descreve o erro."
  remarks: "A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Data.Services.Client.DataServiceQueryException>.</xref:System.Data.Services.Client.DataServiceQueryException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema."
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  syntax:
    content: protected DataServiceQueryException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A ser adicionado."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A ser adicionado."
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String,Exception)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;> </xref> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção."
  remarks: "Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade retorna o mesmo valor que é passado para o construtor, ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Data.Services.Client.DataServiceQueryException>.</xref:System.Data.Services.Client.DataServiceQueryException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DataServiceQueryException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema."
    - id: innerException
      type: System.Exception
      description: "A exceção que é a causa da exceção atual. Se o `innerException` parâmetro não é <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual é gerada em um <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que manipula a exceção interna."
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  id: '#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)'
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: DataServiceQueryException(String,Exception,QueryOperationResponse)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  type: Constructor
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Data.Services.Client.DataServiceQueryException&quot;> </xref> classe."
  syntax:
    content: public DataServiceQueryException (string message, Exception innerException, System.Data.Services.Client.QueryOperationResponse response);
    parameters:
    - id: message
      type: System.String
      description: "O valor de cadeia de caracteres que contém a mensagem de erro."
    - id: innerException
      type: System.Exception
      description: "O objeto de exceção interna."
    - id: response
      type: System.Data.Services.Client.QueryOperationResponse
      description: "O <xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref> objeto."
  overload: System.Data.Services.Client.DataServiceQueryException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Data.Services.Client.DataServiceQueryException.Response
  id: Response
  parent: System.Data.Services.Client.DataServiceQueryException
  langs:
  - csharp
  name: Response
  nameWithType: DataServiceQueryException.Response
  fullName: System.Data.Services.Client.DataServiceQueryException.Response
  type: Property
  assemblies:
  - System.Data.Services.Client
  namespace: System.Data.Services.Client
  summary: "Obtém o <xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref> que indica os resultados de exceção."
  syntax:
    content: public System.Data.Services.Client.QueryOperationResponse Response { get; }
    return:
      type: System.Data.Services.Client.QueryOperationResponse
      description: "Um <xref href=&quot;System.Data.Services.Client.QueryOperationResponse&quot;> </xref> objeto que indica os resultados de exceção."
  overload: System.Data.Services.Client.DataServiceQueryException.Response*
  exceptions: []
  platform:
  - net462
references:
- uid: System.InvalidOperationException
  isExternal: false
  name: System.InvalidOperationException
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException()
  nameWithType: DataServiceQueryException.DataServiceQueryException()
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException()
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(SerializationInfo,StreamingContext)
  nameWithType: DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String,Exception)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor(System.String,System.Exception,System.Data.Services.Client.QueryOperationResponse)
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException(String,Exception,QueryOperationResponse)
  nameWithType: DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
  fullName: System.Data.Services.Client.DataServiceQueryException.DataServiceQueryException(String,Exception,QueryOperationResponse)
- uid: System.Data.Services.Client.QueryOperationResponse
  parent: System.Data.Services.Client
  isExternal: false
  name: QueryOperationResponse
  nameWithType: QueryOperationResponse
  fullName: System.Data.Services.Client.QueryOperationResponse
- uid: System.Data.Services.Client.DataServiceQueryException.Response
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: Response
  nameWithType: DataServiceQueryException.Response
  fullName: System.Data.Services.Client.DataServiceQueryException.Response
- uid: System.Data.Services.Client.DataServiceQueryException.#ctor*
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: DataServiceQueryException
  nameWithType: DataServiceQueryException.DataServiceQueryException
- uid: System.Data.Services.Client.DataServiceQueryException.Response*
  parent: System.Data.Services.Client.DataServiceQueryException
  isExternal: false
  name: Response
  nameWithType: DataServiceQueryException.Response
